Actions/Sign/Sign.ps1

```diff
@@ -0,0 +1,61 @@
+param(
+    [Parameter(HelpMessage = "Azure Key Vault URI.", Mandatory = $true)]
+    [string]$AzureKeyVaultURI,
+    [Parameter(HelpMessage = "Azure Key Vault Client ID.", Mandatory = $true)]
+    [string]$AzureKeyVaultClientID,
+    [Parameter(HelpMessage = "Azure Key Vault Client Secret.", Mandatory = $true)]
+    [string]$AzureKeyVaultClientSecret,
+    [Parameter(HelpMessage = "Azure Key Vault Tenant ID.", Mandatory = $true)]
+    [string]$AzureKeyVaultTenantID,
+    [Parameter(HelpMessage = "Azure Key Vault Certificate Name.", Mandatory = $true)]
+    [string]$AzureKeyVaultCertificateName,
+    [Parameter(HelpMessage = "Path to the folder containing the files to sign.", Mandatory = $true)]
+    [String]$PathToFiles,
+    [Parameter(HelpMessage = "File extensions to sign.", Mandatory = $false)]
+    [String[]]$FileExtensionsToSign = @(".app"),
+    [Parameter(HelpMessage = "Timestamp service.", Mandatory = $false)]
+    [string]$TimestampService = "http://timestamp.digicert.com",
+    [Parameter(HelpMessage = "Timestamp digest algorithm.", Mandatory = $false)]
+    [string]$TimestampDigest = "sha256",
+    [Parameter(HelpMessage = "File digest algorithm.", Mandatory = $false)]
+    [string]$FileDigest = "sha256"
+)
+
+$ErrorActionPreference = "Stop"
+Set-StrictMode -Version 2.0
+$telemetryScope = $null
+$bcContainerHelperPath = $null
+
+# IMPORTANT: No code that can fail should be outside the try/catch
+
+try {
+    . (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
+
+    $BcContainerHelperPath = DownloadAndImportBcContainerHelper -baseFolder $ENV:GITHUB_WORKSPACE 
+
+    Register-NavSip
```

Comment:
This looks like it fits better into another step of the action.

Reply:
Not sure I see a need for this. I personally prefer to have the code in Powershell files rather than embedded in yaml

Reply:
Then why is installing the signing tool a separate task, embedded in the yaml? 😉
It's a one-liner as well: `dotnet tool install --global AzureSignTool --version 4.0.1`

My point is not to have it directly in the yaml, but to achieve good visual separation in the logs (GitHub Actions do that pretty well when there are multiple steps):
-- Install signing tool
-- Register NAV SIP
-- Sign files